sds_data_manager/lambda_code/SDSCode/api_lambdas/query_api.py

```diff
@@ -14,7 +14,8 @@
 logger.setLevel(logging.INFO)
 
 
-def lambda_handler(event, context):
+# Too many conditional branches
```

Comment:
I've been running into this problem in my CoDICE pipeline, I am going to steal this code 😅